integration/go.mod

```diff
@@ -355,3 +355,5 @@ require (
 replace github.com/onflow/flow-go => ../
 
 replace github.com/onflow/flow-go/insecure => ../insecure
+
+replace github.com/onflow/flow-emulator v0.54.1-0.20231110220143-28061d9b37e7 => github.com/AndriiDiachuk/flow-emulator v0.0.0-20231128135645-f5cda716a283
```

Comment:
⚠️  This is fine to temporarily test changes in a dependency as part of development or on a feature branch, but I'm not sure we should merge `replace`s pointing to a remote repo outside of the `onflow` org to `master`. 

Can we remove this `replace` statement and instead add a `require` statement for the version we need, referencing the `onflow/flow-emulator` repo? (Maybe as part of https://github.com/onflow/flow-go/pull/4957.)

In this case, it seems like the `flow-go` changes were backward-incompatible, but the [`flow-emulator` changes](https://github.com/onflow/flow-emulator/pull/514) were backward-compatible, so merging the `flow-emulator` change (without dependency modifications) first would save us some steps.

In cases when both are backward-incompatible, I suggest we keep dependency changes on a branch until we are at least referencing a commit/tag in the `onflow` org version of the repo. 

cc @peterargue @durkmurder

Reply:
@jordanschalm Thanks for comment. I decided to do this kind of thing because of problems with `github.com/onflow/flow-go/crypto` version. In the `flow-go` on `master` version of `crypto` is `v0.24.9` while in the `flow-emulator` on `master`  `crypto` became `v0.24.10`. Because of this integration tests were failing. In my PR(https://github.com/onflow/flow-emulator/pull/514) I downgraded version of `crypto` to be the same as on the` flow-go` `master`. I tried to create branch from the last commit on flow-emulator where the `crypto` version was `v0.24.9`, but there were issues with other dependencies. So if it's possible to merge my PR on `flow-emulator` with `v0.24.9` `crypto` I will remove `replace` statement. If you have other options, will glad to hear them.